### PR TITLE
1371 swat bug

### PR DESCRIPTION
**Pull request by jsadler2**
@pkdash please review. This is implementing the post_metadata_update receiver for SWAT Model Instance.

### CODE REVIEW

**Comment by pkdash** [comment]
@jsadler2 It seems there are some pep/pylint issues to be resolved.

**Comment by jsadler2** [comment]
@pkdash thanks. it's good to go now.

File: setup.cfg [hunk 1]
```diff
@@ -219,6 +219,7 @@ exclude=
   hs_script_resource/tests/__init__.py,
   hs_script_resource/tests/test_script_resource.py,
   hs_swat_modelinstance/admin.py,
+  hs_swat_modelinstance/apps.py,
   hs_swat_modelinstance/forms.py,
```
Comment (pkdash) [inline setup.cfg]: These three lines related to swat model instance should be removed.

Comment (jsadler2) [inline setup.cfg]: Which four lines? All of the lines except apps.py?

**Comment by pkdash** [comment]
@jsadler2 There are still pylint errors.

**Comment by jsadler2** [comment]
@pkdash those have been resolved now.

File: hs_swat_modelinstance/receivers.py [hunk 2]
```diff
@@ -1,27 +1,48 @@
...
 
-@receiver(pre_metadata_element_update, sender=SWATModelInstanceResource)
+
+@receiver(pre_metadata_element_update, sender=swat_models.SWATModelInstanceResource)
 def metadata_element_pre_update_handler(sender, **kwargs):
     return _process_metadata_update_create(**kwargs)
 
+
+@receiver(post_metadata_element_update, sender=swat_models.SWATModelInstanceResource)
+def check_element_exist(sender, **kwargs):
```
Comment (pkdash) [inline hs_swat_modelinstance/receivers.py]: Is there any existing unit test that covers this function?

Comment (pkdash) [inline hs_swat_modelinstance/receivers.py]: I assume this is the only new code in this PR.

File: hs_swat_modelinstance/models.py [hunk 3]
```diff
@@ -325,7 +347,7 @@ def metadata(self):
     @classmethod
     def get_supported_upload_file_types(cls):
         # all file types are supported
-        return ('.*')
+        return '.*'
```
Comment (jsadler2) [inline hs_swat_modelinstance/models.py]: yes. it works.

File: hs_swat_modelinstance/receivers.py (hunk 2 above)
Comment (jsadler2) [inline hs_swat_modelinstance/receivers.py]: This is the only new code in the PR. No there is no unit test that covers this but we could add this later, possibly in a separate issue. Could you point us to code that tests receivers? We don't know how to do that.

Comment (pkdash) [inline hs_swat_modelinstance/receivers.py]: Here is an example of testing a view function. The testing of the view function should cover test for the function in the receivers:

https://github.com/hydroshare/hydroshare/blob/develop/hs_app_timeseries/tests/test_update_sqlite_file_view_function.py

In order to test your receiver function you have to write a unit test for the view function in he_core that updates metadata. For now if you like create an issue to do it after this PR.

**Comment by pkdash** [comment]
+1

**Pull request merged**